src/container/mount.rs

```diff
@@ -185,7 +184,5 @@ pub fn mount_run(run_dir: &Path) -> Result<(), String> {
         .size_bytes(100 << 20)
         .mode(0o755)
         .mount().map_err(|e| format!("{}", e))?;
-    try_msg!(Dir::new(&run_dir.join("shm")).mode(0o1777).create(),
-        "Error creating /vagga/root/run/shm: {err}");
```

Comment:
In the line above we mount absolutely new and clean tmpfs for `/run` so it doesn't matter whether original image ships that directory entry or not.